src/themes/appletv/theme.css

```diff
@@ -62,7 +62,14 @@ html {
     background: #f0f0f0;
 }
 
-.paper-icon-button-light:hover:not(:disabled) {
+@media (hover: hover) {
```

Comment:
This isn't supported on older Chrome versions (Used in Tizen and WebOS), so it needs to be checked in some TV emulators:

https://caniuse.com/#feat=mdn-css_at-rules_media_hover

I have no time to do it now, though.

Reply:
This shouldn't matter though since TV's shouldn't support `hover`.

https://developer.samsung.com/smarttv/develop/specifications/web-engine-specifications.html#media-queries-level-4

Instead, when focused on buttons, it will be in an `focused` state, still being highlighted. I didn't test this on an emulator but used a user string for < Tizen 3.0

Reply:
> This shouldn't matter though since TV's shouldn't support `hover`.

LG's Magic Remote is essentially a mouse pointer and we have to eventually support it properly :)

Reply:
@Camc314 looks like you are trying to fix #1833
I've posted on Matrix: https://medium.com/@mezoistvan/finally-a-css-only-solution-to-hover-on-touchscreens-c498af39c31c
And tried this on iPhone - works.
There is also a link to a test site: https://hover-pointer-media-query.glitch.me/
webOS 1.2 (~Tizen 2.3) doesn't support `hover` and `pointer` - tested on that site. Probably webOS 2 (~Tizen 2.4) doesn't support either.

There is also a JS approach (link in article).

Mouse autofocus enabled for Tizen in `mouseManager`, so I always have focus state. But this is pretty annoying with `scrollManger`.

Reply:
Ok, thanks, I have added `(pointer: fine)` to the media query.

My only concern is for those TV's Pre `webOS TV 3` or `Tizen 2.3` as [level 4 media queries](https://www.w3.org/TR/css3-mediaqueries/) aren't supported there

Reply:
I'll take your word for it.